addons/skin.estouchy/xml/Includes.xml

```diff
@@ -1822,6 +1844,22 @@
 				<textoffsetx>20</textoffsetx>
 			</control>
 			<control type="radiobutton" id="110">
+				<description>Hide Games</description>
+				<width>$PARAM[panel-width]</width>
+				<height>80</height>
+				<aligny>center</aligny>
+				<font>font24_title</font>
+				<textcolor>white</textcolor>
+				<texturefocus>list_focus.png</texturefocus>
+				<texturenofocus></texturenofocus>
+				<pulseonselect>false</pulseonselect>
+				<label>$LOCALIZE[15016]</label>
+				<onclick>Skin.ToggleSetting(HideHomeButtonGames)</onclick>
+				<selected>!Skin.HasSetting(HideHomeButtonGames)</selected>
+				<enable>System.GetBool(gamesgeneral.enable)</enable>
```

Comment:
if game support is disabled, this button gets disabled & the home menu button gets hidden.